pkg/agent/plugin/workloadattestor/k8s/sigstore/sigstore.go

```diff
@@ -186,6 +186,8 @@ func (s *sigstoreImpl) SelectorValuesFromSignature(signature oci.Signature) (*Se
 		return nil, fmt.Errorf("error getting signature provider: %w", errors.New("empty issuer"))
 	}
 
+	// This filter is already done on get signature call (since cosign 2.0),
+	// So it response must no longer contains signatures from unallowed subjects
```

Comment:
```suggestion
	// This filter is already done when getting the signature (since cosign 2.0),
	// so its response must not contain signatures from disallowed subjects.
```